stdlib/Dates/src/types.jl

```diff
@@ -83,7 +83,20 @@ abstract type Calendar <: AbstractTime end
 # ISOCalendar provides interpretation rules for UTInstants to civil date and time parts
 struct ISOCalendar <: Calendar end
 
+"""
+    TimeZone
+
+Geographic zone generally based on longitude determining what the time is at a certain location.
+Some time zones observe daylight savings (eg EST -> EDT).
```

Comment:
I think it'd be helpful to add a mention of the [TimeZones.jl](https://github.com/JuliaTime/TimeZones.jl) package here; if people are looking at `TimeZone` docs, they most likely need the package anyway.